CapitalGainsImporter.js

```diff
@@ -139,10 +139,10 @@ CKI.inputRowData = function(dataSource, row) {
 			var formIndex = rowResp.formIndex;
 			var rowIndex = rowResp.rowIndex;
 
-			var ht = element.querySelector('[name="capitalGains['+formIndex+'].holdingType"]');
-			ht.value = data.holdingType;
+			var bt = element.querySelector('[name="capitalGains['+formIndex+'].belongsTo"]');
```

Comment:
I think this section should only apply when the taxpayer is filing married filing together. This column doesn't show up on single/married filing separately.

Reply:
And it looks like this crashes when the column doesn't exist.